### DISCUSSION TITLE
Hydro Project

### DISCUSSION THREAD

**Discussion by jealy411** [python-hydro/pyro2#101] [Show and tell]
Afternoon! This is Jordan Ealy, first-year graduate student at the University of Maryland, College Park! This semester in our hydro/ISM class, we had term projects which involved simulating various situations and pyro was recommended to me by our professor, Benedikt Diemer! I just finished my write-up and saw under citations that you'd like to know what people were doing with your program. It was super helpful and easy to use!

**Comment by zingale** [comment]
Hi Jordan, thanks for sharing.  I'm glad you found the code useful.
